### DISCUSSION TITLE
Update history without running a command

### DISCUSSION THREAD

**Discussion by madprops** [fish-shell/fish-shell#10773] [Q&A] [answered]
Usually I want to have the last 2 commands on up arrow to be something in particular.

Afaik the only way to update this quickly is by running the commands.

Is there a quick way to up arrow and select an item to be last in history without running it?

I know I can edit the file, wondering if there's something quick like ctrl+enter or such.

**Comment by faho** [comment, marked as answer]
#4506 / #10439 adds `history append`, which you could probably bind.

(to be released in fish 4.0)
